src/utils/SoonExpiringPrescription.ts

```diff
@@ -0,0 +1,27 @@
+import { Prescription } from '../types'
+
+/**
+ * Function to find the prescription with the closest future end date to today
+ * @param prescriptions
+ * @returns
+ */
+export const getSoonExpiringPrescription = (prescriptions: Prescription[]): Prescription | null => {
```

Comment:
I think we have that function in the backend already

Reply:
the cote backend function returns true or false 
it would be necessary to go back up in the prescriptions directly a tag to know which color to display.

Reply:
All this is doing is doing, just returning the prescription that's about to expire the first?
This can be done simply with a lodash.orderBy().
But it should be done in the backend to me

Reply:
https://github.com/mynotif/mynotif-backend/blob/f131d7e/src/nurse/models.py
Then you could add an orderBy at backend level potentially